static/.gitignore

```diff
@@ -1,2 +1,10 @@
 min/
 source-map/
+/js/bundle.js
```

Comment:
Hmm, annoyingly it looks like 3f5d0e6 also didn't do anything to actually delete the file -- it's still lying around in my checkout.

Well, temporary confusion (people seeing this show up as an unknown file in their `git status`) is better than permanent confusion (a puzzling path in the ignore file), and I don't want to give this issue the time to have something remove this file in a way I'm confident is 100% robust. So I'll just take this path out of the ignore file anyway.